home/ryans.py

```diff
@@ -4,12 +4,14 @@
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.chrome.options import Options
 import os
+import time
+
 
 def scrape_ryans(query):
     # Initialize the Selenium WebDriver
     options = Options()
     options.headless = True
-    options.binary_location = os.environ['BROWSER']
+    #options.binary_location = os.environ['BROWSER']
```

Comment:
Why have you commented out this line? This would imply that we want to run the scrapping through the Chrome browser.

Reply:
I have noticed that the Brave browser is running slowly for me that's why I use Chrome. I think that implementing a queue system could potentially lead to better results instead of using Brave Browser.

Reply:
Yeah, that's on my todo list! 😊 

But let's keep the brave as an option, how about checking if the `BROWSER` environment variable is set or not. 

```python
if "BROWSER" in os.environ:
    options.binary_location = os.environ['BROWSER']
```

Reply:
Even after I use the Brave browser, it's not very fast. It may take a little less time than before but it is still very slow.

Reply:
Can you try the current main branch? I have merged @ripa0's multiprocessing code, so it should be much faster now